backend/app/crud/config/version.py

```
@@ -65,13 +65,11 @@ def create_or_raise(self, version_create: ConfigVersionUpdate) -> ConfigVersion:
             validated_blob = ConfigBlob.model_validate(merged_config)
         except ValidationError as e:
             logger.error(
-                f"[ConfigVersionCrud.create_from_partial] Validation failed | "
-                f"{{'config_id': '{self.config_id}', 'error': '{str(e)}'}}"
-            )
-            raise HTTPException(
-                status_code=400,
-                detail=f"Invalid config after merge: {str(e)}",
+                f"[ConfigVersionCrud.create_or_raise] Validation failed | "
+                f"{{'config_id': '{self.config_id}', 'error_count': {len(e.errors())}, "
+                f"'fields': {['.'.join(str(l) for l in err['loc']) for err in e.errors()]}}}"
```

Ruff (0.15.6)

[error] Ambiguous variable name: `l`

(E741)